test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1

```diff
@@ -6,25 +6,25 @@ Describe "Add-Content cmdlet tests" -Tags "CI" {
 
   Context "Add-Content should actually add content" {
     It "should Add-Content to testdrive:\$file1" {
-      $result=add-content -path testdrive:\$file1 -value "ExpectedContent" -passthru
-      $result| Should -BeExactly "ExpectedContent"
+      $result = Add-Content -Path testdrive:\$file1 -Value "ExpectedContent" -PassThru
```

Comment:
Please replace `testdrive:` with `$testdrive` in the file.

Reply:
about_TestDrive.help.txt specifies using TestDrive in this manner.

```
EXAMPLE
    function Add-Footer($path, $footer) {
        Add-Content $path -Value $footer
    }

    Describe "Add-Footer" {
        $testPath="TestDrive:\test.txt"
        Set-Content $testPath -value "my test text."
        Add-Footer $testPath "-Footer"
        $result = Get-Content $testPath

        It "adds a footer" {
            (-join $result).Should.Be("my test text.-Footer")
        }
    }
```

https://github.com/pester/Pester/blob/master/en-US/about_TestDrive.help.txt

Reply:
It is our common pattern. We use this in most files. This is more safe in some cases.
By the way unfortunately the Pester documentation is poorly updated.

Reply:
Pester documentation doesn’t have to follow our patterns of TestDrive usage. :)

I think, let’s codify this in some form, or just use it in accordance with common sense, like does it work in a specific case or not.

Reply:
I don't personally have a strong opinion on this.  Although it should at least be capitalized as `TestDrive:`.  cc @JamesWTruher

Reply:
Fixed.